process-data/flows.mdx

```diff
@@ -35,29 +35,26 @@ To set up a flow configuration:
     If you only specify the name of the dataset in the query, Axiom routes all events to the destination.
     </Note>
 
-4. Click **Preview** to check whether the query you specified transforms your data as desired. The **Input event** section displays the original data stored in Axiom. The **Output event** section displays the transformed data that Axiom sends to the destination. The original data in the Axiom dataset isn’t affected by the transformation.
-5. In the **Destination** section, click **Add a destination**, and then select the destination where you want to route data. 
-6. In the top right, click **Create**.
+1. Click **Preview** to check whether the query you specified transforms your data as desired. The **Input event** section displays the original data stored in Axiom. The **Output event** section displays the transformed data that Axiom sends to the destination. The original data in the Axiom dataset isn’t affected by the transformation.
+1. In the **Destination** section, click **Add a destination**, and then select the destination where you want to route data. 
+1. In the top right, click **Create**.
+1. Select one of the following flow types:
```

Comment:
I think we can leave this step out? Once they've clicked create they have succeeded in creating a flow configuration

Reply:
True, updated as optional step, thanks!